internal/plugins/ansible/v1/scaffolds/internal/templates/config/kdefault/kustomization.go

```diff
@@ -39,7 +39,9 @@ func (f *Kustomization) SetTemplateDefaults() error {
 
 	f.TemplateBody = kustomizeTemplate
 
-	f.IfExistsAction = machinery.Error
+	// For Anible/Helm is no supported webhooks then, we customize
```

Comment:
I think this change can be reverted.

Reply:
Ah nevermind, the base kustomization.yaml is configured with stuff an ansible operator (and helm operator) can't really use out of the box.